### PR TITLE
KEP changes

### PR DESCRIPTION
**Pull request by jeffvance**
+ deleted "Provisioner Secrets" section and all refs to it.
+ BucketClass is no longer used in any brownfield use case.
+ the `protocol` descriptions in the BR and Bucket instance are consistent.

### CODE REVIEW

File: keps/sig-storage/1979-object-storage-support/README.md [hunk 1]
```diff
@@ -163,18 +163,18 @@ status:
...
 1. `bucketAvailable`: if true the bucket has been provisioned. If false then the bucket has not been provisioned and is unable to be accessed.
 
 #### Bucket
 
-A cluster-scoped custom resource representing the abstraction of a single backend bucket. A `Bucket` instance stores enough identifying information so that drivers can accurately target the backend object store (e.g. needed during a deletion process).  Many of the associated bucket class fields are copied to the `Bucket`. Additionally, endpoint data returned by the driver is copied to the `Bucket` by the sidecar.
+A cluster-scoped custom resource representing the abstraction of a single backend bucket. A `Bucket` instance stores enough identifying information so that drivers can accurately target the backend object store (e.g. needed during a deletion process).  All of the associated bucket class fields are copied to the `Bucket`. Additionally, endpoint data returned by the driver is copied to the `Bucket` by the sidecar.
```
Comment (brahmaroutu) [inline keps/sig-storage/1979-object-storage-support/README.md]: what does the endpoint data mean? URI or just bucket name and protocol information?

Comment (jeffvance) [reply keps/sig-storage/1979-object-storage-support/README.md]: This is an original sentence. I agree _endpoint_ is ambiguous. I've been thinking it is URI. Thoughts?

Comment (brahmaroutu) [reply keps/sig-storage/1979-object-storage-support/README.md]: may be ' Information returned by the create bucket call to the backend is populated into the bucket by the sidecar' ?

Comment (brahmaroutu) [reply keps/sig-storage/1979-object-storage-support/README.md]: LGTM

**Pull request merged**